docs/getting-started.md

```diff
@@ -307,8 +308,7 @@ DESCRIPTION="Some short description" ;
 DATA_URL="https://data.catalyst.coop/pudl/core_eia__entity_plants.csv?_size=max" ;
 OUTPUT_DIRECTORY="$DATA_DIR/dataproducts/arfima" ;
 ARTIFACT_YAML="$NAME.yaml" ;
-SERVICE_HOST="localhost" ;
-SERVICE_PORT="2000" ;
+SERVICE_HOST="localhost:8080" ;
```

Comment:
why the change from 2000 to 8080? especially when tests later in the commit still use 2000

Reply:
changed back to 2000